pages/install/conda-install.rst

```diff
@@ -14,39 +14,58 @@
     .. class:: lead
 
       `Anaconda <https://www.anaconda.com/products/individual#Downloads>`__ and
-      `Miniconda <https://docs.conda.io/en/latest/miniconda.html>`__ are Python distributions that include the
-      ``conda`` package manager, which can be used to install Cantera. Both distributions are
-      available for Linux, macOS, and Windows. Note that installing Cantera using Conda will only
-      provide the Cantera Python module. If you want to use the other Cantera interfaces
-      (to use Cantera from MATLAB, Fortran, C++, or C) then see the
-      :ref:`OS-specific installation options <sec-install>`.
+      `Miniconda <https://docs.conda.io/en/latest/miniconda.html>`__ are Python
+      distributions that include the ``conda`` package manager, which can be used to
+      install Cantera. Both distributions are available for Linux, macOS (Intel and
+      ARM/M1), and Windows. Note that installing Cantera using Conda will only provide
+      the Cantera :ref:`Python <sec-conda-python-interface>` or :ref:`MATLAB
+      <sec-conda-matlab-interface>` interfaces. If you want to use the other Cantera
+      interfaces (to use Cantera from Fortran, C++, or C) then see the :ref:`OS-specific
+      installation options <sec-install>`.
 
 Anaconda and Miniconda both include ``conda``; the difference is that Anaconda includes a large
 number of Python packages that are widely used in scientific applications, while Miniconda is a
 minimal distribution that only includes Python and Conda, although all of the packages available in
 Anaconda can be installed in Miniconda. For more details on how to use conda, see the `conda
 documentation <https://docs.conda.io/projects/conda/en/latest/user-guide/index.html>`__.
 
+Conda can install a large set of packages by default and it is possible to install
+packages such as Cantera that are maintained independently. These additional channels
+from which packages may be obtained are specified by adding the ``--channel`` option in
+the ``install`` or ``create`` commands.
+
+.. _sec-conda-python-interface:
+
+Python interface
+================
+
+Cantera's Python interface is available from two channels:
+
+1. The official ``cantera`` channel. This channel should be used if you installed NumPy
+   from the default channel in conda. This channel also has pre-release versions of
+   Cantera for testing.
+2. The ``conda-forge`` channel. This channel should be used if you installed NumPy from
```

Comment:
Not sure if I understand the difference in NumPy here. What combinations work? (It’s not clear if this is a preference or co-dependence).

Reply:
Compilers and ABI compatibility. It actually applies to the base Python too, that might be a better way to state it, with reference to the Python rather than NumPy. Basically, there _might_ be compatibility problems with any compiled packages if you mix compilers. In practice I've never had any problems, but that's the gist of it. I guess it's really easier to say, use Conda forge for all the packages in an environment.

Reply:
Ok, that's good to know. I guess what I was implying is that more direct instructions may be helpful in this write-up. Something along the lines: *In order to ensure compatibility of compiled code, it is recommended to install NumPy from `conda-forge`.*

Reply:
The `conda-forge` channel also doesn't currently include Windows binaries, right?

Reply:
Yep, that's true too, thanks @speth